packages/compiler-cli/src/ngtsc/typecheck/test/diagnostics_spec.ts

```diff
@@ -12,7 +12,7 @@ import {absoluteFrom, getSourceFileOrError} from '../../file_system';
 import {runInEachFileSystem, TestFile} from '../../file_system/testing';
 import {OptimizeFor, TypeCheckingConfig} from '../api';
 
-import {ngForDeclaration, ngForDts, setup, TestDeclaration} from './test_utils';
+import {ngForDeclaration, ngForDts, setup, TestDeclaration} from '../testing/test_utils';
```

Comment:
Like with other packages, structure `testing` to contain a `src` directory with the actual code, and an `index.ts` file (at the same level as `src`) that re-exports the public utility functions. That way the import will be

```suggestion
import {ngForDeclaration, ngForDts, setup, TestDeclaration} from '../testing';
```